src/java/picard/sam/DuplicationMetrics.java

```diff
@@ -41,10 +41,13 @@
      */
     public long UNPAIRED_READS_EXAMINED;
 
-    /** The number of mapped read pairs examined. */
+    /** The number of reads that examined that were either Secondary or Supplementary */
+    public long SECONDARY_OR_SUPPLEMENTARY_READS;
```

Comment:
two things:
1. is this not `_EXAMINED`?  I noticed you changed it back at some point during the many test fixup commits.  Nonetheless it's `UNMAPPED_READS` not `UNMAPPED_READS_EXAMINED` so your call.
2. I would prefer this to go after `READ_PAIRS_EXAMINED`